app/messages.py

```diff
@@ -203,7 +203,8 @@
 USER_SUCCESSFULLY_UPDATED = {"message": "User was updated successfully."}
 PASSWORD_SUCCESSFULLY_UPDATED = {"message": "Password was updated "
                                  "successfully"}
```

Comment:
can this be accommodated on the previous line?

Reply:
And add a period at the end of `successfully` :see_no_evil: